src/vs/editor/contrib/gotoSymbol/peek/referencesController.ts

```diff
@@ -278,6 +290,30 @@ function withController(accessor: ServicesAccessor, fn: (controller: ReferencesC
 	}
 }
 
+KeybindingsRegistry.registerCommandAndKeybindingRule({
+	id: 'changeFocus',
+	weight: KeybindingWeight.WorkbenchContrib + 50,
+	primary: KeyCode.F2,
+	when: ctxReferenceSearchVisible,
+	handler(accessor) {
+		withController(accessor, controller => {
+			controller.changeFocusBetweenPreviewAndReferences();
+		});
+	}
+});
+
+KeybindingsRegistry.registerCommandAndKeybindingRule({
+	id: 'changeFocusFromEmbeddedEditor',
+	weight: KeybindingWeight.EditorContrib + 50,
+	primary: KeyCode.F2,
```

Comment:
👎 for using `F2`. Given the amount of VS Code users there certainly will be some that want to rename. Leaving unbound or using something like ctrl+left and ctrl+right.